v3.0/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -133,20 +133,14 @@ spec:
             # Cluster type to identify the deployment type
             - name: CLUSTER_TYPE
               value: "k8s,bgp"
+            # Disable file logging so `kubectl logs` works.
```

Comment:
Oh, interesting. I thought that @2ffs2nns was saying that the v3.0 manifest was the one that worked....but only after switching out the version numbers for master.

Reply:
nope, master works, but v3.0 did not. This PR copies master->v3.0.